deployment/ansible/roles/driver.app/tasks/main.yml

```diff
@@ -17,15 +17,7 @@
   command: >
     docker build
       -f {{ root_app_dir }}/Dockerfile.base
-      -t quay.io/azavea/driver-app:latest
-      {{ root_app_dir }}
-  when: developing
-
-- name: Build application Docker image
-  command: >
-    docker build
-      -f {{ root_app_dir }}/Dockerfile.development
```

Comment:
Is `Dockerfile.development` no longer used? If so, we should remove it.

Reply:
Also, I see a reference to `driver-app:latest` a few lines down (line 25), that may need to be modified, but I'm not certain.

Reply:
Yeah, I think it needs to reference `docker_image_tag`.